Sources/Subprocess/Platforms/Subprocess+Darwin.swift

```diff
@@ -192,67 +202,121 @@ extension Configuration {
                 defer {
                     posix_spawn_file_actions_destroy(&fileActions)
                 }
+
                 // Input
                 var result: Int32 = -1
-                if let inputRead = inputPipe.readFileDescriptor {
-                    result = posix_spawn_file_actions_adddup2(&fileActions, inputRead.platformDescriptor, 0)
+                if inputReadFileDescriptor != nil {
```

Comment:
Suggest using `if let inputReadFileDescriptor {` so you don't need the force unwrap below.

Reply:
Unfortunately this pattern, along with `if let a = a` is not compatible with `~Copyable` types. Writing `if let inputReadFileDescriptor` would have consumed `inputReadFileDescriptor` so the line below would be double consume.